app/controllers/orga/seasons_controller.rb

```diff
@@ -82,6 +72,10 @@ def season_params
     )
   end
 
+  def phase
+    params[:phase].to_sym if phase.in? PHASES
```

Comment:
As I said in a previous comment, since I think we can be sure to **always** be on a Ruby version > 2.2, we'll have GC for symbols. Therefore, no problem with DoS when converting strings to symbols in this case.

I'd vote for removing the `if` condition here. I also cannot imagine it working *(`PHASES` is defined in a different namespace, would be `Season::PhaseSwitcher::PHASES`)*... plus I don't think it makes huge sense to first check wheather the param is whitelisted, if not, pass on nil and then check again if it's whitelisted.

So, I'd suggest to just go with:
```ruby
def phase
  params[:phase].to_sym
end
```
and then to the whole whitelist stuff in the model where the actual business logic lies.